packages/ui/src/components/_utils/hooks/useBreakpoints.tsx

```diff
@@ -25,7 +25,7 @@ export function useBreakpoints (refreshOnChange = true): ScreenMap {
 
             return () => ResponsiveObserve.unsubscribe(token)
         }
-    }, [])
+    }, [forceUpdate, refreshOnChange])
```

Comment:
Very strange logic inside forceUpdate, how does it work using the counter, why reducer instead of state ?